core/data-plane/data-plane-framework/src/main/java/org/eclipse/dataspaceconnector/dataplane/framework/DataPlaneFrameworkExtension.java

```diff
@@ -129,6 +135,16 @@ public void shutdown() {
         }
     }
 
+    @NotNull
+    private TransferProcessApiClient registerTransferProcessClient(ServiceExtensionContext context) {
+        if (transferProcessApiClient != null) {
+            return transferProcessApiClient;
+        }
+        var noopClient = new NoopTransferProcessClient();
+        context.registerService(TransferProcessApiClient.class, noopClient);
```

Comment:
So we are missing the @Provides annotation at the top of the class then ? 

Couldn't we use a default Provider for the NoopTransferClient? 

@ndr-brt  what do you think?

Reply:
@bscholtes1A right. To be coherent with the convention, I'd create a `DataPlaneDefaultServices` where that noop `TransferProcessClient` is provided by default